### ISSUE TITLE
Add a global throttling rule

### ISSUE THREAD

**Issue by laurijamsa** [ruuvi/ruuvi-network-serverless#32]
If a Ruuvi Gateway sends data too often to Ruuvi Network, discard some of the messages. The minimum allowed time between messages to be stored on the database should be configurable (in milliseconds).

The time limit can be global.

**Comment by Muhwu** [comment]
WAF is one option in front of the API gateway: https://docs.aws.amazon.com/waf/latest/developerguide/what-is-aws-waf.html

Alternatively, introducing Redis to the stack for really fast in-memory database might be a good idea as some sort of caching solution for faster requests could work. According to Massimiliano, there are some limits that can be bypassed with SNS topics so when we're thinking about alerting, Redis based solution might be more efficient. This will, however, imbue a cost of *tens* of dollars per month (and can be scaled up in the unlikely scenario of tremendous volume).

In short:
* WAF can do IP based throttling, not sure about costs - but it is basically built for this
* Redis is more manual but we can use it in other application areas as well

There are definitely other options as well and our data pruning thing currently simply pokes dynamo instead of in-memory db like Redis to determine whether to store data in the sparse table.
